reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/PublishWithSelector.kt

```diff
@@ -0,0 +1,16 @@
+package com.badoo.reaktive.observable
```

Comment:
Let's move it to `Publish.kt` file? It is small, has same name, so it will be easier to find it. Also we won't generate additional class for JVM.